app/models/doctor.rb

```diff
@@ -0,0 +1,7 @@
+class Doctor < ApplicationRecord
+  # Relationships will be added here when we add more tables
+  # Validations will be added here
+  validates :name, presence: true
```

Comment:
Could you please try to merge to one line of code  after you set up all your attributes?